### DISCUSSION TITLE
Checked status for Check Picker

### DISCUSSION THREAD

**Discussion by trananhvan1102vn** [rsuite/rsuite#3701] [Q&A] [answered]
Hi, I want to implement some logic to filter data, but I need to know if user uncheck or check an option in component Check Picker

I found that this is not passing down to onSelect event. Please tell me if there is any reason behind it.

![image](https://github.com/rsuite/rsuite/assets/76468917/b5d6209f-cd45-42a0-ac09-041cb8bb52f9)

**Comment by simonguo** [comment, marked as answer]
Hi @trananhvan1102vn  Isn't the `onChange` event what you need? Filter your data by the value of the `onChange` callback.
